### ISSUE TITLE
Pet Treasure Finding (Sniffing)

### ISSUE THREAD

**Issue by tara81** [MyPetORG/MyPet#722]
Pet Pooping

=-=-=

Pets will poop every 10 minutes, They will drop an item beside them. Only pets owned by players will poop. The item dropped by all pets is specified by a command, that is held in the admins hands. Admin command only. Colorful names and lores supported.

http://mc.pwing.net/wp-content/uploads/2015/09/PetPoop2.png

/petadmin poop hand <amount>

**Comment by tara81** [comment]
(I thought you would have a laugh at this one) hehe !

**Comment by xXKeyleXx** [comment]
They will poop cocoa beans. And then players will backe cookies with them.

**Comment by tara81** [comment]
lmao.

**Comment by tara81** [comment]
If you thought this was a good idea, maybe it would be better if the pet only pooped once every 30 minutes .  And the item dropepd could be random from the list offered by an admin.  That way a player could get anything from a diamond to a cobblestone ;D

**Comment by xXKeyleXx** [comment]
But I don't know when it will happen :P

**Comment by tara81** [comment]
and there could be a chat message sent to the player"
Your pet Skunky left something behind ...
or
Your pet Skunky has made a mess !
or 
Your pet Skunky has pooped on the grass. Oh dear.

**Comment by tara81** [comment]
yeah thats fine , I'm guessing u got bugs to fix first and other requests to do ;D

esp that nasty pet range skill bug.  Its painful to watch your pet slowly walk to the monster to shoot it hahahahaha

**Comment by xXKeyleXx** [comment]
I think something like a treasure skill would suit the plugin better but the idea is the same.

**Comment by tara81** [comment]
You pet has sniffed out some treasure !

**Comment by xXKeyleXx** [comment]
Exactly :+1: 

I already fixed the ranged bug but haven't uploaded a fix yet because I changed most of the permissions and some config option names. Next upload will be the proper release.

**Comment by tara81** [comment]
damn your fast lol.  I hope the premium plugin is doing well for you =)

**Comment by xXKeyleXx** [comment]
Way better than I expected xD

**Comment by xXKeyleXx** [comment]
From a joke to a proper idea xD

**Comment by tara81** [comment]
Lol .  I make my cows poop on my server amd make my players collect it for fertilizer ...

**Comment by tara81** [comment]
I put tons of jokes in my server quests lol.

**Comment by FatherWh0** [comment]
You might consider letting them drop .. 'poop?'.. their death drops. You could steal... err.. borrow mcmmo's treasures.yml table for shaking. This would allow server owners to tweak the drops as they see fit. Also it would save you the time and effort of making up your own loot tables.
http://pastebin.com/5g1jmfPF

**Comment by UserDerezzed** [comment]
Added Sniff skill in MyPet 4.0.0-alpha-27
